src/m365/spo/commands/listitem/listitem-batch-remove.ts

```diff
@@ -167,7 +169,10 @@ class SpoListItemBatchRemoveCommand extends SpoCommand {
         if (args.options.filePath) {
           const csvContent = fs.readFileSync(args.options.filePath, 'utf-8');
           const jsonContent = formatting.parseCsvToJson(csvContent);
-          idsToRemove = jsonContent.map((item: { ID: string }) => item['ID']);
+
+          const idRow = Object.keys(jsonContent[0]).find(key => key.toLowerCase() === 'id');
+
+          idsToRemove = idRow && jsonContent.map((item: any) => item[idRow]);
```

Comment:
Since we already check if there's a valid ID column in the validators, I don't think the check here is really needed.


```suggestion
          idsToRemove = jsonContent.map((item: any) => item[idRow]);
```